### DISCUSSION TITLE
How do you build the HTML documentation?

### DISCUSSION THREAD

**Discussion by marek22k** [chris2511/xca#585] [Q&A] [answered]
Hello,

I would like to know how to build the HTML documentation?

**Comment by chris2511** [comment, marked as answer]
XCA uses [sphinx](https://www.sphinx-doc.org/en/master/man/sphinx-build.html).

In the xca directory the target "sphinx-html" builds the doc:
`cmake -B build && cmake --build build -t sphinx-html`

Internally, the documentation in "doc/rst" is used and the documentation of the commandline options, the export formats
and SQL Schema are automatically generated from the sources.

The sphinx-build command is called here https://github.com/chris2511/xca/blob/cb1f02502d80eeaa1b886a80559d8ea6653bf16d/doc/CMakeLists.txt#L93

**Reply by marek22k** [reply]
Thank you for your answer. I was now able to successfully build a Zeal/Dash compatible documentation: https://mark22k.codeberg.page/zeal-docs/@xca/xca.xml
